setup/README.md

```diff
@@ -1,87 +1,92 @@
 
-### 2. Configure the environment variables
+## Deployment Steps
 
-Copy the `.env.example` file to `.env` and modify the values as needed for your specific deployment:
+### 1. Open Azure Cloud Shell and access the Otto repository:
 
-```cmd
-copy .env.example .env
-```
+You have two options:
 
-### 3. Build and connect to the setup container
+**Option A: Clone the repository (if you haven't done so before)**
 
-The setup container contains all the necessary tools and dependencies to deploy the Otto infrastructure. You can choose to open the container in Visual Studio Code or directly in the Docker terminal.
-
-**Option 1**: Open in Visual Studio Code
-
-Open Visual Studio Code and follow the steps below:
+```bash
+git clone https://github.com/justicecanada/otto.git
+cd otto/setup
+```
 
-- Open the Command Palette (Ctrl+Shift+P)
-- Type "Dev Containers: Open Folder in Container" and select the option
-- Select the repo's `setup` folder
+**Option B: Update an existing repository**
 
-This process may take a few minutes the first time as it builds the container image. Once it completes, open a new terminal in Visual Studio Code to run the commands.
+If you've previously cloned the Otto repository, navigate to the Otto directory and pull the latest changes:
 
-**Option 2**: Open in Docker
+```bash
+cd otto
+git pull
+cd setup
+```
 
-If you are able to login to Azure from within a container using `--use-device-code`, you can run the following command:
+### 2. Create infrastructure using Terraform:
 
 ```bash
-docker-compose up -d
-docker exec -it otto-setup-container /bin/bash
+bash run_terraform.sh
 ```
 
-## Create Infrastructure
-
-### 1. Create infrastructure using Terraform
+Note: You'll be prompted to either input the `ENTRA-CLIENT-SECRET` or use the value if it exists in the Key Vault already.
 
-Review and apply any necessary infrastructure changes.
+### 3. Deploy the AKS cluster:
 
 ```bash
-bash deploy_terraform.sh
+bash run_k8s.sh
 ```
-**Note:** The `CLIENT_SECRET` is not stored in the `.tfvars` file for security reasons. You can provide it interactively when prompted.
 
-### 2. Configure the OpenAI Capacity
+Important: Ensure the Otto image exists in the container registry before deploying the AKS cluster. (The image is built and pushed as part of the Continuous Integration pipeline.)
 
-After the Terraform script has completed deploying the OpenAI resource and its default model deployments, the capacity for each model deployment needs to be adjusted manually. This is necessary to ensure that the deployments have sufficient capacity to handle the expected workload.
+# Appendix: Development Team Guide
 
-**Note:** Instances of OpenAI within the same subscription share the same capacity pool. Ensure that the total capacity across all deployments does not exceed the subscription's capacity limit.
+For developers who want to emulate and test the scripts within a Docker container:
```

Comment:
What do you mean by "emulate"? Modify?
And they aren't really "testing" the scripts, they will be run just the same as if running in Cloud Shell, right? I think this is important because the statement makes it sound like it is totally safe to run them outside Cloud Shell and nothing can go wrong.

Reply:
They can be run from outside the Cloud Shell though.